test/spec/directives/block.spec.js

```diff
@@ -199,6 +199,16 @@ describe('Directive: blocks', () => {
 
       expect($scope.params.includeInternational).not.toBeDefined();
     });
+
+    it('checks the campus database if a answer is present on page load', () => {
```

Comment:
I think that a test like this would be better. It mocks the network call to return `[]` (i.e. no campuses) and checks that the answer was set back to blank.

```js
    it('checks the campus database if a answer is present on page load', () => {
      $httpBackend
        .whenGET('campuses/SFSU?includeInternational=true&limit=15')
        .respond(() => [200, []]);

      $scope.answer = { value: 'SFSU' };
      $compile('<campus-question></campus-question>')($scope);

      $httpBackend.flush();
      $httpBackend.verifyNoOutstandingExpectation();
      $httpBackend.verifyNoOutstandingRequest();

      expect($scope.answer.value).toBe('');
    });
```

The `forms the searchCampuses params correctly` and `doesn't add includeInternational` tests could also use network mocking instead of checking $scope.params, but that doesn't need to be done in this PR.